content/sensu-enterprise/3.6/changelog.md

```diff
@@ -58,13 +59,23 @@ _NOTE: Sensu Enterprise is built on Sensu Core. Sensu Core changes are documente
 - [Enterprise 1.14.0 Release Notes](#enterprise-v1-14-0)
 - [Enterprise 1.13.0 Release Notes](#enterprise-v1-13-0)
 
+## Enterprise 3.6.2 Release Notes {#enterprise-v3-6-2}
+
+**October 16, 2019** &mdash; Sensu Enterprise version 3.6.2 has been
+released and is available for immediate download. Please note the
+following improvements:
+
+### CHANGES {#enterprise-v3-6-2-changes}
+
+- **BUGFIX**: Fixed handling of closed connections, no longer producing Java::JavaLang::NullPointerException errors.
```

Comment:
```suggestion
- **BUGFIX**: Fixed closed connections handling so that they no longer produce Java::JavaLang::NullPointerException errors.
```

Reply:
```suggestion
- **BUGFIX**: Fixed connection handling so that unexpectedly closed connections no longer produce Java::JavaLang::NullPointerException errors.
```

Reply:
@hillaryfraley how about this latest suggestion? ☝️